### PR TITLE
GUI:add default button

### LINKED ISSUE #208: GUI: Add Reset to Defaults button
This would allow user to reset the settings back to defaults

>change the settings dictionary in `GUI.py`

### PR DESCRIPTION
**Pull request by lohani2280**
fixes #208

### CODE REVIEW

**Comment by abhsag24** [comment]
Hi thank you for your Patch make sure you have the right commit structure that is 
```
tag: Shortlog

body
Fixes <issue number/url>
```

File: uiplib/gui/settingsTab.py [hunk 1]
```diff
@@ -140,6 +147,17 @@ def handle_settings(appObj):
     update_settings(appObj.new_settings)
 
 
+def default_settings(appObj):
+    """Method to update the settings file to default setting."""
+    appObj.new_settings['pics-folder'] = DEFAULT_SETTINGS['pics-folder']
```
Comment (abhsag24) [inline uiplib/gui/settingsTab.py]: infact why a new function?

File: uiplib/settings.py [hunk 2]
```diff
@@ -6,6 +6,16 @@
 HOME_DIR = os.path.join(os.path.expanduser("~"), '.uip')
 NUMBER_OF_IMAGES_TO_PARSE = 15
 DEFAULT_PICS_FOLDER = os.path.join(HOME_DIR, 'pics')
+DEFAULT_SETTINGS = {'timeout': 30*60,
```
Comment (abhsag24) [inline uiplib/settings.py]: hey we decalare this dict again on `setup.py` can you make the changes so we dont write it there and just use it there as well?

Comment (lohani2280) [inline uiplib/settings.py]: yeah sure

File: setup.py [hunk 3]
```diff
@@ -12,7 +12,7 @@
 from uiplib.settings import (HOME_DIR,
                              DEFAULT_PICS_FOLDER,
                              NUMBER_OF_IMAGES_TO_PARSE,
-                             settings_file_path)
+                             settings_file_path, DEFAULT_SETTINGS)
```
Comment (abhsag24) [inline setup.py]: Have all the constants be together in newlines meaning something like this: 
```
.....,
NUMBER_OF_IMAGES_TO_PARSE,
DEFAULT_SETTINGS,
settings_file_path)
```

File: setup.py [hunk 4]
```diff
@@ -45,16 +45,8 @@ def get_requirements(filename):
...
-                 'no-of-images': NUMBER_OF_IMAGES_TO_PARSE,
-                 'pics-folder': DEFAULT_PICS_FOLDER,
-                 'website': ['https://unsplash.com/new',
-                             'https://www.reddit.com/r/wallpapers/',
-                             'https://www.reddit.com/r/wallpaper/',
-                             'https://www.reddit.com/r/EarthPorn/',
-                             'https://www.reddit.com/r/VillagePorn/',
-                             'https://www.reddit.com/r/pics/',
-                             'https://api.desktoppr.co/1/wallpapers', ]}
+    file_data = DEFAULT_SETTINGS
```
Comment (abhsag24) [inline setup.py]: nice and neat

File: uiplib/gui/settingsTab.py [hunk 5]
```diff
@@ -81,7 +82,13 @@ def create_settings_tab(appObj):
     apply_button = Button(mainFrame,
                           text="Apply",
                           command=lambda: handle_settings(appObj))
-    apply_button.grid(row=6, column=1, pady=20, sticky=W)
+    apply_button.grid(row=6, column=1, pady=10, sticky=W)
+
+    # Reset
+    default_button = Button(mainFrame,
+                            text="Default",
+                            command=lambda: update_settings(DEFAULT_SETTINGS))
```
Comment (abhsag24) [inline uiplib/gui/settingsTab.py]: or if it is too much effort we can have a new issue for it.

Comment (lohani2280) [inline uiplib/gui/settingsTab.py]: yeah sure i am on it.

**Comment by nkprince007** [comment]
Could you rebase and push again @lohani2280?

**Comment by abhsag24** [comment]
@lohani2280 i checkoud out your PR can you make the default button in the same row as the Apply button? and maybe call it Reset to defaults?

**Comment by lohani2280** [comment]
@abhsag24  i made changes as u suggested . kindly have a look .

**Comment by abhsag24** [comment]
@lohani2280 awesome stuff! merging :D

**Comment by abhsag24** [comment]
@lohani2280  merged awesome suff! :D

**Pull request merged**